CustomHeaderPlugin.inc.php

```diff
@@ -109,30 +109,33 @@ function displayTemplateHook($hookName, $params) {
 		if (!$this->injected) {
 			$this->injected = true;
 			$templateMgr =& $params[0];
+			$templateResource =& $params[1];
 			$request = Application::get()->getRequest();
 			$context = $request->getContext();
-			$templateMgr->addHeader('custom', $this->getSetting($context?$context->getId():CONTEXT_ID_NONE, 'content'));
+			if($templateResource != "plugins-1-plugins-generic-pdfJsViewer-generic-pdfJsViewer:submissionGalley.tpl")
+				$templateMgr->addHeader('custom', $this->getSetting($context?$context->getId():CONTEXT_ID_NONE, 'content'));
```

Comment:
I think this check for existence on `$context` is probably borrowed from somewhere else in the codebase, right? It should always be safe to call `$context->getId()` here, rather than the ternary expression; `$context` should always be defined.